runtime/doc/options.txt

```diff
@@ -2135,10 +2135,10 @@ A jump table for the options with a short description can be found at |Q_op|.
 	   fuzzy    Enable |fuzzy-matching| for completion candidates. This
 		    allows for more flexible and intuitive matching, where
 		    characters can be skipped and matches can be found even
-		    if the exact sequence is not typed. Note: This option
+		    if the exact sequence is not typed.  Note: This option
```

Comment:
just curious. why are there two spaces here? I've seen this happen elsewhere. 🤔

Reply:
I don't know :) It's for consistency with a descriptions of other options.

Reply:
Yeah, the documentation tends to prefer 2 spaces after ending a sentence. I believe this was typical writing style for monospaced documents in older days and I believe was Brams preferred style. See e.g. here: https://writing.stackexchange.com/a/1347